demos/crossroad_camera_demo/cpp/main.cpp

```diff
@@ -564,8 +567,8 @@ int main(int argc, char *argv[]) {
 
         /** Start inference & calc performance **/
         typedef std::chrono::duration<double, std::ratio<1, 1000>> ms;
-        auto total_t0 = std::chrono::high_resolution_clock::now();
 
+        auto startTime = std::chrono::steady_clock::now();
```

Comment:
What if we hide it into static method `PerformanceMetric::getTime()`?
Or in pair methods `PerformanceMetric::start()` `PerformanceMetric::stop()`?

Reply:
First option would be just wrapper over `std::chrono::steady_clock::now();`, do we need it?
What for the second option, `PerformanceMetrics` class needs only start time stamp, end time it measures inside `update()` function, so adding pair methods would require refactoring.

Reply:
Sorry for delay. Wrapping the `std::chrono::steady_clock::now()` we give control for timing directly to `PerformanceMetric` class, so the timer could be easily changed any moment

Reply:
Why you expect it to be changed any moment?

Reply:
Ok. It also removes the `#include <chrono>` from demo.

Reply:
Right now it's aligned with python `PerformanceMetrics`, so if we really want to add wrapper, we will have to change python implementation as well. I think it's better be done with different PR,  because all demos would be affected, and this PR is already large one.